bin/windows-toolset.ps1

```diff
@@ -171,3 +159,21 @@ cd $toolsDir\${opensslPkg}
 Invoke-External { & 7za x "$toolsDir\${opensslPkg}.tar" }
 
 cd $toolsDir
+# Download ruby
+Write-Host "Downloading http://buildsources.delivery.puppetlabs.net/windows/ruby/${rubyPkg}.7z"
+(New-Object net.webclient).DownloadFile("http://buildsources.delivery.puppetlabs.net/windows/ruby/${rubyPkg}.7z", "$toolsDir\${rubyPkg}.7z")
+Invoke-External { & 7za x "$toolsDir\${rubyPkg}.7z" | FIND /V "ing " }
+
+$env:PATH = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
+if ($arch -eq 32) {
+    $env:PATH = "C:\tools\mingw32\bin;" + $env:PATH
+} else {
+    $env:PATH = "C:\tools\mingw64\bin;" + $env:PATH
+}
+$env:PATH += [Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd" + ";" + "$toolsDir\$rubyPkg\bin" + ";" + "$toolsDir\$opensslPkg\bin"
```

Comment:
Extraneous string additions can be written as:

```powershell
$env:PATH += [Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd;$toolsDir\$rubyPkg\bin;$toolsDir\$opensslPkg\bin"
```

If you were trying to break up for readability, then

```powershell
@([Environment]::GetFolderPath('ProgramFiles') + "\Git\cmd",
"$toolsDir\$rubyPkg\bin",
"$toolsDir\$opensslPkg\bin") | 
  % { $Env:PATH += ";$($_)" }